ci-operator/config/openstack-k8s-operators/neutron-operator/openstack-k8s-operators-neutron-operator-master.yaml

```diff
@@ -76,6 +80,19 @@ tests:
   optional: true
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: neutron-operator-build-deploy-tempest
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h20m0s
+    version: "4.10"
+  optional: true
+  steps:
+    env:
+      TEMPEST_REGEX: tempest.api.network.*.v3|neutron_tempest_plugin
```

Comment:
remove neutron_tempest_plugin, from the latest result, it seems it doesn't work.
tempest.api.network doesn't have v3, if you check https://opendev.org/openstack/tempest/src/branch/master/tempest/api/network/ the regex follow the directory structure, so it is tempest.api.network, there is a admin directory in network, but it doesn't have a v3 directory, that's the reason the tempest.api.network did not run

Reply:
done